packages/tsx-editor/src/transpiler/exampleTransform.ts

```diff
@@ -43,8 +50,17 @@ export function transformExample(example: string, id: string) {
    * error since the import is not supported.
    */
   while ((temp = importPattern.exec(example))) {
-    if (!/office-ui-fabric-react/.test(temp[0])) {
-      output.error = 'error: unsupported imports';
+    if (/\/exampleData/.test(temp[0])) {
```

Comment:
You could probably get rid of the repeated code by doing something similar to what I did in codepenTransform.ts.

```ts
const exampleDataFiles = {
  exampleData: /*the raw-loader import*/,
  TestImages: /*import*/,
  PeopleExampleData: /*import*/
};
```

Then loop over the entries in here:
```
let foundExampleImport = false;
for (const filename of Object.keys(exampleDataFiles)) {
  if (temp[0].indexOf('/' + filename) !== -1) {
    example += `\n${exampleDataFiles[filename]}\n`;
    example = example.replace(temp[0], '');
    foundExampleImport = true;
    break;
  }
}
```